typescript/sdk/src/index.ts

```diff
@@ -60,7 +60,7 @@ export {
   CoreFactories,
   coreFactories,
 } from './core/contracts.js';
-export { HyperlaneLifecyleEvent } from './core/events.js';
+export { HyperlaneLifecycleEvent } from './core/events.js';
```

Comment:
Renaming the export without fixing the thing it's referring to will break the SDK. Did you try building this before opening your PR?